test/companies/schema/company_test.exs

```diff
@@ -18,13 +16,13 @@ defmodule Companies.Schema.CompanyTest do
     industry_id: 12346
   }
 
-  describe "company changeset" do
-    test "company/2 validates with the correct parameters" do
+  describe "changeset/2" do
+    test "returns a valid changeset for correct parameters" do
       changeset = Company.changeset(%Company{}, @valid_company_params)
       assert changeset.valid?
     end
 
-    test "errors out if given invalid params" do
```

Comment:
It's not always possible but let's try to keep the phrasing similar been identical tests.